cmake/TestScript.cmake

```diff
@@ -3,6 +3,4 @@ set (CTEST_BINARY_DIRECTORY /home/markus/Projekte/Elektra/current/build)
 set (CTEST_COMMAND ctest)
 set ($ENV{srcdir} /home/markus/Projekte/Elektra/current/build/tests)
 
-set (MemoryCheckCommand /usr/bin/valgrind)
```

Comment:
> again, any specific reason?

The same one as mentioned above: The command `valgrind` is located in `/usr/local/bin/valgrind` on my machine and `make run_memcheck` works if I remove this line.